src/Application/Security/Password.php

```diff
@@ -25,7 +25,13 @@ public static function hash(string $rawPassword): string
      */
     public static function verify(string $rawPassword, Person $person): void
     {
-        if (!password_verify($rawPassword, $person->getPasswordHash())) {
+        $hash = $person->getPasswordHash();
+
+        if ($hash === null) {
```

Comment:
Perhaps we could do `if ($hash === null) || !password_verify($rawPassword, $hash))` to reduce code redundancy?

Reply:
@alihejazi I don't love this idea. I think having separate clauses for this critical functionality is more readable and promotes us eventually getting better test coverage.